ui/apps/platform/src/services/ComplianceProfileService.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'services/instance';
+import qs from 'qs';
+
+import { complianceV2Url } from './ComplianceCommon';
+
+const complianceProfilesBaseUrl = `${complianceV2Url}/profiles`;
+
+export type ComplianceProfileSummary = {
+    name: string;
+    productType: string;
+    description: string;
+    title: string;
+    ruleCount: number;
+    profileVersion: string;
+};
+
+/**
+ * Fetches a list of compliance profile summaries based on the provided cluster IDs.
+ */
+export function listComplianceSummaries(clusterIds): Promise<ComplianceProfileSummary[]> {
```

Comment:
Asking, not necessarily recommending, whether `listComplianceProfileSummaries` helps readers at the call site or whether shorter is clearer.

Reply:
Good question, I think I originally read the proto as `ListComplianceSummaries` not `ListProfileSummaries`. I do however think `listComplianceProfileSummaries` is by far the most clear but at the same time it doesn't align with the proto.

Reply:
With that said, I imagine a lot of these method names don't line up one-to-one. This feels like a good opportunity to do that.

Reply:
Oh, my bad. My eyes must have crossed. I meant align with backend function name, even if it could be better.

Reply:
Having said that, for any endpoints added since 4.4 release, it makes sense to suggest improvement to backend names based on how they read in frontend context, or how endpoint names align with their response type names.

Reply:
I agree